arches/app/views/plugin.py

```diff
@@ -33,13 +33,25 @@ def get(self, request, pluginid=None, slug=None):
             plugin = models.Plugin.objects.get(slug=slug)
         else:
             plugin = models.Plugin.objects.get(pk=pluginid)
+
         if not request.user.has_perm("view_plugin", plugin):
```

Comment:
In a follow-up, we might want to expose this as a config option as well. If my vue app desires to handle login itself, currently, this will redirect you to the arches auth page unnecessarily.

Reply:
Agreed 👍